index.html

```diff
@@ -36,7 +36,7 @@
           <li><a href="#"><i class="fa-solid fa-bell"></i></a></li>
         </ul>
         <div class="user-account">
-          <a href="http://youtube.com" target="_blank"><img src="https://source.unsplash.com/random/?Cryptocurrency/"
+          <a href="#"><img src="https://source.unsplash.com/random/?Cryptocurrency/"
               alt="crypto" /></a>
```

Comment:
🤦‍♂️ As I mentioned earlier, I can still see the previously required changes. If you followed these best practices of semantic tags, you would better improve your code in future projects as well, so go ahead and make those changes to improve your code.👍